test/test.googleauth.ts

```diff
@@ -641,12 +654,12 @@ describe('googleauth', () => {
   });
 
   it('getProjectId should return a new projectId the first time and a cached projectId the second time', async () => {
-    mockEnvVar('GCLOUD_PROJECT', fixedProjectId);
+    mockEnvVar('GCLOUD_PROJECT', STUB_PROJECT);
```

Comment:
Where is `STUB_PROJECT` even defined?  What was wrong with just `fixedProjectId`?

Reply:
In the same file:

https://github.com/googleapis/google-auth-library-nodejs/blob/47905529d225ca76c020f2400a57fae6150e9536/test/test.googleauth.ts#L42-53

It just happened that we had two variables with same values and same meaning.